package.json

```diff
@@ -14,7 +14,7 @@
     "e2e:dev": "./e2e/start-and-run-suite dev",
     "jest": "jest --notify --watch",
     "lint": "eslint . --ext .js,.tsx,.ts --cache",
-    "jest-ci": "mkdir -p reports/junit && export JEST_JUNIT_OUTPUT_DIR=reports/junit && jest --ci --reporters=default --reporters=jest-junit -w ${TEST_MAX_WORKERS:-100%}",
+    "test": "mkdir -p reports/junit && export JEST_JUNIT_OUTPUT_DIR=reports/junit && jest --ci --reporters=default --reporters=jest-junit -w ${TEST_MAX_WORKERS:-100%}",
```

Comment:
Are you sure you want the "test" script to use CI parameters? For example this script passes `--ci` to jest and makes a report directory. I'm not sure that the script should be the same for local testing and CI.

Reply:
Maybe I will do a poll about this but personally I've never used the test command locally. I used the jest command instead. Maybe change this back to jest-ci and rename the jest command to test?

Reply:
I think it's best to follow the convention of having the -ci suffix for CI scripts. If you don't have a non-ci script for local usage, that's fine. I think it doesn't matter if it's called jest-ci or test-ci FWIW.